README.md

```diff
@@ -7,7 +7,8 @@ Core delivery platform Node.js Frontend Template.
     - [Node.js](#nodejs)
   - [API](#api)
   - [Local development](#local-development)
-    - [Setup](#setup)
+    - [Developing services that use CDP-Uploader](#developing-services-that-use-the-cdp-uploader)
+    - [Setup](#setup-for-developing-cdp-uploader)
     - [Development](#development)
```

Comment:
A few things missing from the TOC - there are generators online as long as things are not sensitive that you are submitting - https://ecotrust-canada.github.io/markdown-toc/